{{cookiecutter.project_slug}}/docker-compose.shared.base-images.yml

```
@@ -1,7 +1,7 @@
 version: '3.3'
 services:
   db:
-    image: postgres:11
+    image: postgres:12
```

```suggestion
    image: postgres:11
```

Let's leave the upgrade to a separate PR to keep the scope clear and narrow.